recipes/octo-keygen-cpp/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package LANGUAGES CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(${PROJECT_NAME} ../test_package/test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE CONAN_PKG::octo-keygen-cpp)
```

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE octo-keygen-cpp::octo-keygen-cpp)
```
CONAN_PKG will almost deprecated

---

fixed :)